custom/rch/models.py

```diff
@@ -0,0 +1,207 @@
+import datetime
+from django.db import models
+from custom.rch.utils import fetch_beneficiaries_records, MOTHER_DATA_TYPE, CHILD_DATA_TYPE
+
+STATE_DISTRICT_MAPPING = {
+    '28': [
+        '523'
+    ]
+}
+
+
+class RCHRecord(models.Model):
+    Caste = models.CharField(null=True, max_length=255)
+    Created_On = models.DateTimeField(null=True)
+    District_ID = models.PositiveSmallIntegerField(null=True)
+    District_Name = models.CharField(null=True, max_length=255)
+    HealthBlock_Name = models.CharField(null=True, max_length=255)
+    HealthBlock_ID = models.IntegerField(null=True)
+    Taluka_ID = models.IntegerField(null=True)
+    Taluka_Name = models.CharField(null=True, max_length=255)
+    MDDS_TalukaID = models.IntegerField(null=True)
+    MDDS_DistrictID = models.PositiveSmallIntegerField(null=True)
+    PHC_Name = models.CharField(null=True, max_length=255)
+    PHC_ID = models.IntegerField(null=True)
+    Subcentre_Name = models.CharField(null=True, max_length=255)
+    Village_ID = models.IntegerField(null=True)
+    MDDS_VillageID = models.IntegerField(null=True)
+    Mobile_no = models.BigIntegerField(null=True)
+    Religion = models.CharField(null=True, max_length=255)
+    State_Name = models.CharField(null=True, max_length=255)
+    StateID = models.PositiveSmallIntegerField(null=True)
+    Updated_On = models.DateTimeField(null=True)
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    last_modified = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+    @classmethod
+    def update_beneficiaries(cls, beneficiary_type):
+        date_str = str(datetime.date.fromordinal(datetime.date.today().toordinal() - 1))
+        for state_id in STATE_DISTRICT_MAPPING:
+            for district_id in STATE_DISTRICT_MAPPING[state_id]:
+                records = fetch_beneficiaries_records(date_str, date_str, state_id, beneficiary_type, district_id)
+                for record in records:
+                    rch_beneficiary = cls()
+                    for prop in record:
+                        setattr(rch_beneficiary, prop.keys()[0], prop.values()[0])
```

Comment:
To make this a bit safer you could get all the fields on the model and make sure the props are a valid field. You could also do validation with it if you want:
```python
valid_fields = {field.name: field for field in cls._meta.fields}
field = prop.keys()[0]
if field not in valid_fields:
    # skip or do something
if isinstance(valid_fields[field], models.TimeField):
    # check that it is correct format
```